src/Packetery/Core/Validator/Order.php

```diff
@@ -117,7 +117,7 @@ public function validate( Entity\Order $order ): array {
 	 * @return string
 	 */
 	private function getTranslation( string $key ): string {
-		if ( empty( $this->translations[ $key ] ) ) {
+		if ( ! isset( $this->translations[ $key ] ) ) {
```

Comment:
`return $this->translations[ $key ] ?? $key;` ?

Reply:
yees (y)